datafusion/core/benches/sql_planner.rs

```diff
@@ -60,6 +60,44 @@ pub fn create_table_provider(column_prefix: &str, num_columns: usize) -> Arc<Mem
     MemTable::try_new(schema, vec![]).map(Arc::new).unwrap()
 }
 
+pub fn create_tpch_schemas() -> [(String, Schema); 2] {
+    let lineitem_schema = Schema::new(vec![
+        Field::new("l_orderkey", DataType::Int32, false),
+        Field::new("l_partkey", DataType::Int32, false),
+        Field::new("l_suppkey", DataType::Int32, false),
+        Field::new("l_linenumber", DataType::Int32, false),
+        Field::new("l_quantity", DataType::Float64, false),
+        Field::new("l_extendedprice", DataType::Float64, false),
+        Field::new("l_discount", DataType::Float64, false),
+        Field::new("l_tax", DataType::Float64, false),
```

Comment:
It may not be important for this benchmark, but financial amounts should be decimal rather than float. We have these schema definitions already in https://github.com/apache/arrow-datafusion/blob/main/benchmarks/src/tpch/mod.rs#L45 that should match the TPC-H specification.

Reply:
I must have messed something up when i was cleaning up and creating PR, i had actually copied from that exact location.  Thx for catching, will fix.